conf/config.neon

```diff
@@ -76,6 +76,7 @@ parameters:
 	checkNullables: false
 	checkThisOnly: true
 	checkUnionTypes: false
+	checkBenevolentUnionTypes: false
```

Comment:
should it be true on bleedingEdge?

Reply:
CheckUnionTypes is not true on bleedingEdge, so checkBenevolentUnionTypes which is stricter shouldn't be true too.

Reply:
`checkBenevolentUnionTypes` will never be true by default, that's why it's "benevolent".

Reply:
But I'm kinda worried about someone using the config
```
checkUnionTypes: false
checkBenevolentUnionTypes: true
```
what should be the behavior ?
- checking both ?
- checking none ?
- only checking the benevolent ?
- throwing an error ?